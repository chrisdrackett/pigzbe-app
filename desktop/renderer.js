import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
