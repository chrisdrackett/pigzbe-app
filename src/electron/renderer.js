import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
