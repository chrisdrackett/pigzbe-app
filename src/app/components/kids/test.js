import React from 'react';
import renderer from 'react-test-renderer';
import Kids from './';

const props = {
    dispatch: () => {},
    kids: [],
};

describe('Kids', () => {
    test('renders correctly', () => {
        renderer.create(<Kids {...props}/>);
    });
});
