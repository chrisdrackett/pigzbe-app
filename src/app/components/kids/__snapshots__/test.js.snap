// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kids renders correctly 1`] = `
<View
  style={
    Object {
      "paddingBottom": 20,
      "paddingTop": 25,
    }
  }
>
  <Text
    style={
      Object {
        "color": "rgb(255, 255, 255)",
        "fontFamily": "Poppins",
        "fontSize": 16,
        "fontWeight": "bold",
        "marginBottom": 12,
        "textAlign": "center",
      }
    }
  >
    Kids wallets
  </Text>
  <View
    style={
      Object {
        "paddingLeft": "6.25%",
        "paddingRight": "6.25%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "stretch",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "rgb(255, 255, 255)",
            "borderColor": "rgb(255, 255, 255)",
            "borderRadius": 22.5,
            "borderWidth": 1,
            "marginBottom": 10,
            "maxWidth": 400,
            "opacity": 1,
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "color": "rgb(255, 255, 255)",
                "fontFamily": "Poppins",
                "fontSize": 14,
                "fontWeight": "bold",
                "height": 45,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 12,
                "textAlign": "center",
              },
              Object {
                "color": "rgb(0, 50, 120)",
              },
              null,
              undefined,
            ]
          }
        >
          Add My Children
        </Text>
      </View>
    </View>
  </View>
</View>
`;
